resources/js/Pages/Collections/View.tsx

```diff
@@ -272,8 +272,14 @@ const CollectionsView = ({
                                         </div>
                                     </div>
 
-                                    {nfts.paginated.data.length === 0 && query !== "" ? (
-                                        <EmptyBlock>{t("pages.collections.search.no_results")}</EmptyBlock>
+                                    {nfts.paginated.data.length === 0 ? (
+                                        query !== "" ? (
+                                            <EmptyBlock>{t("pages.collections.search.no_results")}</EmptyBlock>
+                                        ) : (
+                                            <EmptyBlock>
+                                                {t("pages.collections.search.no_results_with_filters")}
```

Comment:
this will also apply when using the "show my collection" toggle and if you don't own any NFTs. Will discuss if that's wanted or if that should show something else instead

Reply:
for the case where the user toggles and does not own any, we should show `You don't own any NFTs in this collection.` as message instead